examples/demo/config/development.yaml

```diff
@@ -156,3 +156,35 @@ auth:
     expiration: 604800 # 7 days
 # </snip>
 
+# Scheduler Jobs Configuration
+# <snip id="configuration-scheduler">
+scheduler:
+  # Location of shipping the command stdout and stderr.
+  output: stdout
+  # A list of jobs to be scheduled.
+  jobs:
+      # The name of the job.
+    - name: "Run command"
+      # The type of job.
+      shell:
+        command: "echo loco >> ./scheduler.txt"
+      # The cron expression defining the job's schedule.
+      cron: "*/1 * * * * *"
+      output: silent
+      tags:
```

Comment:
can we do `tags: ['base', 'infra']` in YAML example?

Reply:
Done. https://github.com/loco-rs/loco/pull/735/commits/667ce389fc4284b7c2220e0428f51e0c34a35bc3